results/README.md

```diff
@@ -46,15 +68,18 @@ $ go get -u github.com/golang/protobuf/protoc-gen-go
 ```
 $ go generate ./proto/
 ```
-4. Install the sqlite3
+
+### Teardown
+
+To delete associated resources:
 
 ```
-$ apt-get install sqlite3
+kubectl delete -f config/
 ```
 
-5. Create results.db and taskrun table under the cmd/api folder
+And don't forget to delete the secret:
 
 ```
-$ sqlite3
-$ sqilte> .read results.sql
```

Comment:
Let's keep both local and remote instructions.

Reply:
With this change there won't be any `sqlite` at all, and there's only really the mode where you connect to a remote mysql database (either running in-cluster or in Cloud SQL)